repondeur/tests/models/test_article.py

```diff
@@ -77,6 +88,72 @@ def test_types(self, lecture_an):
             titre < motion < chapitre < section < sous_section < article < annexe < vide
         )
 
+    @pytest.mark.parametrize(
+        "before, after",
+        list(
+            pairwise(
+                [
+                    "11",
+                    "11 bis A",
+                    "11 bis",
+                    "11 ter",
+                    "11 quater",
+                    "11 quinquies",
+                    "11 sexies",
+                    "12 AA",
+                    "12 AB",
+                    "12 A",
+                    "12 BA",
+                    "12 BB",
+                    "12 BC",
+                    "12 BD",
+                    "12 B",
+                    "12 C",
+                    "12 D",
+                    "12",
+                    "12 bis",
+                    "13",
+                    "13 bis",
+                    "13 ter",
+                    "13 quater A",
+                    "13 quater",
+                    "13 quinquies",
+                    "13 sexies",
+                    "14",
+                    "14 bis A",
+                    "14 bis",
+                    "15",
+                    "16",
+                    "17",
+                    "17 bis AAA",
+                    "17 bis AA",
+                    "17 bis A",
+                    "17 bis BA",
+                    "17 bis BB",
+                    "17 bis B",
+                    "17 bis C",
+                    "17 bis",
+                    "17 ter",
+                ]
+            )
+        ),
+    )
+    def test_andouillette(self, lecture_an, before, after):
```

Comment:
❤️ can't wait for that one to fail, it deserves a dedicated pytest plugin!

Reply:
😋